docs/Home.asciidoc

```diff
@@ -0,0 +1,180 @@
+[[sync-framework-fh.sync]]
+= Sync Framework (fh.sync)
```

Comment:
I don't see the standalone version being referred to as fh.sync. I would see it just being sync, as you would probably do something like `var sync = require('fh-sync')` and not have any 'fh' object